src/models/Settings.js

```diff
@@ -397,9 +397,9 @@ export default Model.extend({
 
   processCreds: function(creds) {
     const processCreds = this.get('processCreds');
-
+    const credentialsAreComplete = creds.username && creds.password;
```

Comment:
why need add this check?

Reply:
See the discussion here: https://github.com/okta/okta-signin-widget/pull/1851#discussion_r616153242

Reply:
I've reverted this change